api/disruption_types_test.go

```diff
@@ -291,3 +292,127 @@ var _ = Describe("DisruptionStatus.RemoveTargets Test", func() {
 		})
 	})
 })
+
+var _ = Describe("Disruption Annotations Tests", func() {
```

Comment:
Can you move this test in the `v1beta1/disruption_types_test.go` and in the `Describe("Disruption", func() {` ?